docs/charts/bar.md

```diff
@@ -88,7 +88,7 @@ the color of the bars is generally set this way.
 | ---- | ----
 | `clip` | How to clip relative to chartArea. Positive value allows overflow, negative value clips that many pixels inside chartArea. `0` = clip at chartArea. Clipping can also be configured per side: `clip: {left: 5, top: false, right: -2, bottom: 0}`
 | `label` | The label for the dataset which appears in the legend and tooltips.
-| `order` | The drawing order of dataset. Also affects order for stacking, tooltip, and legend.
```

Comment:
personally I prefer the comma here, but just a matter of preference ;-)

Reply:
Omg, how did I manage not to know about `Oxford comma` before!?
This is just how lists are expected to be written in Finnish, and it looked wrong to me.
Its fine by me either way.